python/examples/integrations/writers/Writing_to_WhyLabs.ipynb

```diff
@@ -0,0 +1,482 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Writing Profiles to WhyLabs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/whylabs/whylogs/blob/mainline/python/examples/writers/integrations/Writing_to_WhyLabs.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example, we'll show how to send your logged profiles to your monitoring dashboard at WhyLabs Platform.\n",
+    "We will:\n",
+    "\n",
+    "- Define environment variables with the appropriate Credentials and IDs\n",
+    "- Log data into a profile\n",
+    "- Use the WhyLabs Writer to send the profile to your Project at WhyLabs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
We might want to include a pip install command to pull in the whylabs extra?

```
!pip install whylogs[whylabs]
```

Reply:
Yes, forgot about that.